.github/workflows/release.yml

```
@@ -0,0 +1,65 @@
+          
+          # Create archives
+          zip GitLab-Importer-linux-amd64.zip GitLab-Importer-linux-amd64
+          zip GitLab-Importer-windows-amd64.zip GitLab-Importer-windows-amd64.exe
+          zip GitLab-Importer-darwin-amd64.zip GitLab-Importer-darwin-amd64
+          
+          # Generate checksums
+          sha256sum GitLab-Importer-*.zip > checksums.txt
+
+      - name: Generate Changelog Notes
+        id: changelog
+        uses: mikepenz/release-changelog-builder-action@v2
```

actionlint

the runner of "mikepenz/release-changelog-builder-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)